modules/ROOT/pages/work-with-metadata.adoc

```diff
@@ -0,0 +1,60 @@
+= Work With Metadata in Studio
+
+Anypoint Studio 7.10 infers the metadata of your current flow so that, when you use a Flow Reference component to reference another flow, you can set the metadata from your current flow as custom metadata for your referenced flow. By automatically detecting the actual metadata of your existing flow, Studio enables you to propagate the metadata in your flow to the flow you are referencing as the expected metadata.
```

Comment:
We should remove the `as the expected metadata` because we propagate actual or expected based on different conditions, might be confusing